django/core/signing.py

```diff
@@ -135,18 +144,19 @@ def dumps(obj, key=None, salt='django.core.signing', serializer=JSONSerializer,
     return TimestampSigner(key, salt=salt).sign_object(obj, serializer=serializer, compress=compress)
 
 
-def loads(s, key=None, salt='django.core.signing', serializer=JSONSerializer, max_age=None):
+def loads(s, key=None, salt='django.core.signing', serializer=JSONSerializer, max_age=None, read_keys=None):
```

Comment:
It would probably be `load_keys` but what about `key_fallbacks` instead to have the same naming as we have elsewhere?

Reply:
I'm good with `key_fallbacks`.